### PR TITLE
feat: wagmi version updated

### PR DESCRIPTION
**Pull request by manneredboor**
### Description

Wagmi version updated

### Checklist:

- [x] Checked the changes locally.

### CODE REVIEW

File: providers/sdk-legacy.tsx [hunk 1]
```diff
@@ -1,86 +1,65 @@
...
-
-    const providerMainnetRpc = getStaticRpcBatchProvider(
-      mainnet.id,
-      rpc[mainnet.id],
-      mainnet.id,
-      pollingInterval,
-    );
-    providerRpc.on('debug', onRpcProviderError);
-    providerMainnetRpc.on('debug', onRpcProviderError);
+  const providerRpc = getStaticRpcBatchProvider(
```
Comment (Jeday) [inline providers/sdk-legacy.tsx:50]: Why useMemo is removed?

**Review by Jeday — APPROVED** [review]

File: providers/sdk-legacy.tsx [hunk 2]
```diff
@@ -1,86 +1,65 @@
...
+    chainId,
+    rpc[chainId],
+    0,
+    POLLING_INTERVAL,
+  );
 
-    return { providerRpc, providerMainnetRpc };
-  }, [chainId, pollingInterval, rpc]);
+  const providerMainnetRpc = getStaticRpcBatchProvider(
```
Comment (Jeday) [inline providers/sdk-legacy.tsx:57]: useMemo?

**Review by Jeday — CHANGES_REQUESTED** [review]

File: providers/sdk-legacy.tsx (hunk 1 above)
Comment (manneredboor) [reply providers/sdk-legacy.tsx:50]: The `getStaticRpcBatchProvider` function has a memo inside itself already

**Review by Jeday — APPROVED** [review]

**Review by solidovic — APPROVED** [review]

**Review by DiRaiks — APPROVED** [review]

**Review by alx-khramov — COMMENTED** [review]
LGTM ✅

**Pull request merged**